values-vvp.yaml

```diff
@@ -28,6 +28,8 @@ vvp:
             taskmanager.memory.managed.fraction: 0.0 # no managed memory needed for filesystem statebackend
             high-availability: vvp-kubernetes
             metrics.reporter.prom.class: org.apache.flink.metrics.prometheus.PrometheusReporter
+            execution.checkpointing.interval: 10s
+            execution.checkpointing.externalized-checkpoint-retention: RETAIN_ON_CANCELLATION
```

Comment:
Please check that these configurations are available in the referenced vvp version. :)